HW #3 - Nakhimov Ilya (A copy of the #8 PR)

nakhimov-i opened the pull request:
This is a copy of a December 20 pull request #8 that was accidentally closed.

etray7 commented on splitAndMerge.js:
```diff
@@ -0,0 +1,9 @@
+    
+    for(var key in wordsSeparator){
+      result.push((wordsSeparator[key].split('')).join(sp));
+    }
```
Don't use `for in` for array

nakhimov-i commented:
Kirill, thank you for the code review.

I fixed the bugs: 
1. Rewrote the "convert" function to a more compact form. 
2. Fixed 'let' on 'var' in the "countDown" function. 
3. Replaced 'for in' with the classic array search.
